pages/_en-US/ds-index/forwarders.md

```diff
@@ -52,15 +52,11 @@ If you already have Universal Updater installed on your console, you can skip to
 ### Requirements
 
 - A Nintendo DSi with [Unlaunch](https://dsi.cfw.guide/installing-unlaunch) and [hiyaCFW](../hiyacfw/installing) installed
-- The latest release of [Title Manager for HiyaCFW](https://github.com/JeffRuLz/TMFH/releases)
-- A 64-bit OS
-- The latest release of [Forwarder3-DS](https://www.dropbox.com/s/b9de5ii6vm3dxfn/Forwarder3DS-v2.9.6.zip?dl=0)
-- [Java 8](https://www.java.com/en/download/)
-- **Linux users:** JavaFX
-   - Debian-based: Run [this script](https://gist.githubusercontent.com/puntillol59/7532b6583380baca236dcaf2d8f75b5c/raw/e8b9d193f8b24de941160c7292ec0bb3b997e98e/main.sh)
-   - Arch: `sudo pacman -S java8-openjfx && sudo archlinux-java set java-8-openjdk/jre`
+- The latest release of [NDSForwarder-DSi](https://github.com/lifehackerhansol/NDSForwarder-DSi/releases)
 
 ### Part 1: Getting started
+1. Copy `NDSForwarder.dsi` to your SD card root
+   - This can optionally be installed to hiyaCFW directly using TMFH
```

Comment:
```suggestion
   - This can optionally be installed to hiyaCFW directly using [TMFH](https://github.com/JeffRuLz/TMFH/releases/latest/download/TMFH.0.7.1.zip)
```
do y'all think it'd be better to recommend NTM or TMFH? NTM has a few bug fixes and usability improvements even with SDNAND mode, but there's also the same thing as with Safe NAND Manager where increased promotion of it means increased likelihood someone does something wrong...
If y'all think NTM, alternate suggestion:
```suggestion
   - This can optionally be installed to the DSi Menu using [NTM](https://github.com/Epicpkmn11/NTM/releases/latest/download/NTM.dsi)
```
It might also be worth making a info bubble or part 4 saying you can install the forwarders to SysNAND with NTM.

Reply:
I'm personally going to keep it as a direct installer. If someone wants to use NTM let them use Forwarder3-DS.

Maybe have a separate tab for DSi SysNAND.

Reply:
You can easily install it to SDNAND then just install `sd:/title/00030004/########/content/00000000.app`/back up from SDNAND, install to SysNAND using NTM so it doesn't really need a different section, could even just be an info bubble really.